src/sys/uio.rs

```diff
@@ -19,7 +19,16 @@ pub fn readv(fd: RawFd, iov: &mut [IoVec<&mut [u8]>]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
-#[cfg(target_os = "linux")]
+/// Write bytes to `fd` beginning at `offset` from buffers in `iov` in order
+/// until all buffers have been written or an error occurs. The file offset is
+/// not changed.
+///
+/// See also: `writev` and `pwrite`
```

Comment:
Please link to these functions using [inline links](https://rust-lang-nursery.github.io/api-guidelines/documentation.html#prose-contains-hyperlinks-to-relevant-things-c-link).